l10n_it_intrastat_statement/tests/test_intrastat_statement.py

```diff
@@ -230,6 +241,34 @@ def test_statement_purchase_refund(self):
         self.assertEqual(len(file_lines), 3)
         self.assertSetEqual({len(line) for line in file_lines}, {75, 130, 119})
 
+    def test_statement_purchase_refund_no_compensation(self):
+        bill = self._get_intrastat_computed_bill(date="2018-12-15")
+        self._get_intrastat_computed_bill(date="2019-01-05")
+
+        bill_refund = self._create_move_refund(bill, date="2019-01-10")
+        # This refund will be subtracted from bill
+        bill_refund.update(
+            {
+                "intrastat": True,
+            }
+        )
+        bill_refund.action_post()
+        bill_refund.compute_intrastat_lines()
+
+        statement = self.statement_model.create(
+            {
+                "period_number": bill_refund.invoice_date.month,
+                "fiscalyear": bill_refund.invoice_date.year,
+            }
+        )
+
+        statement.compute_statement()
+
+        # No compensation should happen, we expect both purchase section 1 and 2
```

Comment:
Qui e sopra dove scrivi "we expect..." potresti scrivere una riga di spiegazione sul _perché_ ci aspettiamo un certo comportamento? Almeno un accenno al fatto che il periodo non è quello della dichiarazione aiuterebbe.
Qui o nella docstring del test andrebbe bene

Reply:
> Qui e sopra dove scrivi "we expect..." potresti scrivere una riga di spiegazione sul _perché_ ci aspettiamo un certo comportamento? Almeno un accenno al fatto che il periodo non è quello della dichiarazione aiuterebbe. Qui o nella docstring del test andrebbe bene

Ok! Ora lo faccio